js/views/conversation_view.js

```diff
@@ -350,7 +350,12 @@
             props: getGroupSettingsProp(this.model),
           });
           this.$('.conversation-content-right').append(this.groupSettings.el);
+          this.$('.conversation-content-right').show();
```

Comment:
Nitpick, but you could avoid duplicating this line if you used `if...else` instead of an early return.

Reply:
fixed